app/src/ui/about/about.tsx

```diff
@@ -232,7 +237,7 @@ export class About extends React.Component<IAboutProps, IAboutState> {
           </Row>
           <h2>{name}</h2>
           <p className='no-padding'>
-            Version {version} ({releaseNotesLink})
+            <LinkButton title='Click to copy' className='version-text' onClick={this.onClickVersion}>Version {version}</LinkButton> ({releaseNotesLink})
```

Comment:
I'm not around for the next few weeks but I'll leave my thoughts here because I think this needs some design input.

The fact that this text is now clickable isn't really helpful. Here's what I now see:

<img width="472" src="https://user-images.githubusercontent.com/359239/27717780-40555ad2-5cfc-11e7-93f1-596d67f5718f.png">

And unlike all the other links here, there's no feedback when you do click that it's done anything. You need to hover for a couple of seconds to see the `Click to copy` message.

I think we can do better than this, and I like where the discussion in #1778 ended up.